src/components/project/project-member/project-member.service.ts

```diff
@@ -197,24 +189,20 @@ export class ProjectMemberService {
 
     const query = this.db
       .query()
-      .apply(matchRequestingUser(session))
-      .match([node('node', 'ProjectMember', { id })])
-      .apply(matchPropList)
       .match([
         node('project', 'Project'),
         relation('out', '', 'member', { active: true }),
-        node('', 'ProjectMember', { id }),
+        node('node', 'ProjectMember', { id }),
+        relation('out', '', 'user'),
+        node('user', 'User'),
       ])
-      .with(['project', 'node', 'propList'])
-      .apply(matchMemberRoles(session.userId))
-      .match([node('node'), relation('out', '', 'user'), node('user', 'User')])
```

Comment:
@michaelmarshall Are you aware of performance implications of moving these 3 match statements into one? (as I have in new code).

Reply:
I do not. I've been surprised enough though. Not without testing on a large dataset would I be able to say.